src/ChooseDialect.js

```diff
@@ -0,0 +1,55 @@
+import { List } from "react-native-paper";
+import React from "react";
+import styled from "styled-components/native";
+
+export const ChooseDialect = () => {
+  const dialects = [
+    {
+      name: "広島弁",
+      en: "Hiroshima",
+    },
+    {
+      name: "大阪弁",
+      en: "Osaka",
+    },
+    {
+      name: "京都弁",
+      en: "Kyoto",
+    },
+  ];
+
+  return (
+    <SafeareaContainer>
```

Comment:
Why don't we put `<SafeareaContainer>` in App.js so we don't have to do it on every component. It will just be there for everything.